modular_skyrat/modules/customization/modules/language/selenian.dm

```diff
@@ -0,0 +1,15 @@
+/datum/language/selenian
+	name = "Selenian"
+	desc = "A bastardization of Old World Latin. The language is primarily spoken by the inhabitants of Luna."
```

Comment:
may be worth updating this description a tad, using some of the Cultural Background that's given for Selenian - "bastardization of latin" makes it seem like a pure Sol ripoff instead of something unique; less likely we'll keep it if it's seen as "fucked up version of a language we have" (dont look at ashwalkers)

I personally recommend something along the lines of:
```suggestion
	desc = "A bastardization of Old World Latin, with traces of several other prominent Old Earth languages, mostly those of the powerful Dynasties and corporate forerunners who first reached the stars. The language is primarily spoken by the inhabitants of Luna, though as Lunar progress boomed Selenian spread fairly far."
```